Automattic/wp-calypso
# Gutenberg: only show opt-in flows to Simple Sites

This PR updates the Opt-In components to not render if a site is a Jetpack site. We'll only want to target simple sites until we're ready to support Jetpack flows. Depending on what VIP would like to do, we may need to add an additional check here later.

Simple Site:
![screen shot 2018-11-02 at 10 11 24 am](https://user-images.githubusercontent.com/1270189/47931909-61f4fa80-de8d-11e8-826d-811fdafa2593.png)

Jetpack:
![screen shot 2018-11-02 at 10 51 40 am](https://user-images.githubusercontent.com/1270189/47931920-6a4d3580-de8d-11e8-9eac-0d331f28bce9.png)


#### Changes proposed in this Pull Request

* Moves the feature check to the opt-in components, and adds another render check if the site is a Jetpack site.

#### Testing instructions

In localhost, or on horizon.wordpress.com:
* Navigate to /post
* Select a Simple Site
We should see the opt-in components as expected

In localhost or on horizon.wordpress.com:
* Navigate to /post
* Select a Jetpack Site
We should _not_ see the opt-in visuals.
